examples/xdc-example/src/app/xdc.tx.example.ts

```diff
@@ -0,0 +1,309 @@
+import { TatumXdcSDK } from '@tatumio/xdc'
+
+const xdcSDK = TatumXdcSDK({ apiKey: '75ea3138-d0a1-47df-932e-acb3ee807dab' })
+
+export async function xdcTxWithSignatureIdExample(): Promise<void> {
+  // NATIVE
+  const preparedTransferNativeTransaction = await xdcSDK.transaction.prepare.transferSignedTransaction({
+    to: '0x811DfbFF13ADFBC3Cf653dCc373C03616D3471c9',
+    amount: '10',
+    signatureId: 'cac88687-33ed-4ca1-b1fc-b02986a90975',
+    nonce: 3252345722143,
+    fee: {
+      gasLimit: '53632',
+      gasPrice: '20',
+    },
+  })
+
+  const sentTransferNativeTransaction = await xdcSDK.transaction.send.transferSignedTransaction({
+    to: '0x811DfbFF13ADFBC3Cf653dCc373C03616D3471c9',
+    amount: '10',
+    signatureId: 'cac88687-33ed-4ca1-b1fc-b02986a90975',
+    nonce: 3252345722143,
+    fee: {
+      gasLimit: '53632',
+      gasPrice: '20',
+    },
+  })
+
+  // ERC20(FUNGIBLE TOKEN)
+  const preparedDeployErc20Transaction = await xdcSDK.erc20.prepare.deploySignedTransaction({
```

Comment:
dont use prepare functions in examples, use only send - prepare is for our internal use